skbeam/core/roi.py

```diff
@@ -757,3 +759,65 @@ def auto_find_center_rings(avg_img, sigma=1, no_rings=4, min_samples=3,
         edge_pts_xy = edge_pts_xy[~inliers]
 
     return center, image, radii
+
+def find_center_rings_hough(image, num_rings, rad_range, intensity_thresh, sigma, v_range = None):
+    """
+    Take angled cut or multiple incremented cuts of a image data.
+
+    Parameters
+    ----------
+    image : ndarray
+        Image data
+    num_rings : int
+        Number of candidate circles to be searched for
+    rad_range : array
+        The min, max, and increment for range of candidate radii to be searched
+    intensity_thresh : array
+        Min and max of intensity to be used in canny filter
+    sigma : float
+        Standard deviation applied in canny filter
+    v_range : array
+        Min and max for matplotlib colormap
+
+    Returns
+    -------
+    center_pt : array
+        Average center of the candidate circles
+    radii: array
+        radii of the candidate circles
+
+    """
+
+    low_thresh = intensity_thresh[0]
```

Comment:
A nice pattern for this is:

```py
low_thresh, hi_thresh = intensity_thresh
```

It's more succinct, and more importantly it provides some validation. If the user mistakenly provides, `intensity_thresh=[1, 2, 3]`, the suggestion about would raise an error about an unexpected number of elements in the list (it wants exactly two) whereas the current implementation would silently ignore the extra value.